accelerometer/summariseEpoch.py

```diff
@@ -87,21 +87,24 @@ def getActivitySummary(epochFile, nonWearFile, summary,
 
     # Remove data before/after user specified start/end times
     rows = e.shape[0]
+    tz = pytz.timezone(timeZone)
     if startTime:
-        e = e[e.index >= startTime]
+        localStartTime = tz.localize(startTime)
+        e = e[e.index >= localStartTime]
     if endTime:
-        e = e[e.index <= endTime]
+        localEndTime = tz.localize(endTime)
+        e = e[e.index <= localEndTime]
     # Quit if no data left
     if e.shape[0] == 0:
         print("No rows remaining after start/end time removal")
         print("Previously there were %d rows, now shape: %s" % (rows, str(e.shape)))
         sys.exit(-9)
 
     # Get start & end times
-    startTime = pd.to_datetime(e.index.values[0])
-    endTime = pd.to_datetime(e.index.values[-1])
-    summary['file-startTime'] = startTime.strftime('%Y-%m-%d %H:%M:%S')
-    summary['file-endTime'] = endTime.strftime('%Y-%m-%d %H:%M:%S')
+    startTime = pd.to_datetime(e.index.values[0]).tz_convert(tz)
+    endTime = pd.to_datetime(e.index.values[-1]).tz_convert(tz)
```

Comment:
Is it needed to convert `e.index` again? I think it's already in `tz` time...

Reply:
Thanks for spotting this. Fixed it. Well, I can't show the test result yet since the gt3x test cases haven't been merged yet.